mass/mathstat/toeplitz.py

```diff
@@ -154,14 +154,17 @@ def __precompute_symmetric(self):
 
     def __solve_symmetric(self, y):
```

Comment:
what argument type must y have since it has to have the `ndim` field? An `ndarray`? what number of dimensions does it need in the `y.ndim != 2` case?

Reply:
Good point. I have added type annotations and reworked this file, too.